src/io/flutter/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -24,60 +24,44 @@
 import com.intellij.testFramework.LightVirtualFile;
 import com.intellij.util.BitUtil;
 import com.intellij.util.TimeoutUtil;
-import com.intellij.xdebugger.XDebugProcess;
-import com.intellij.xdebugger.XDebugSession;
-import com.intellij.xdebugger.XDebugSessionListener;
-import com.intellij.xdebugger.XDebuggerBundle;
-import com.intellij.xdebugger.XSourcePosition;
+import com.intellij.xdebugger.*;
```

Comment:
This one is just going to keep changing, depending on who formats it. The AS rules use the expanded form.

Reply:
Ah, sorry.  The actual setting is a little further down:

![image](https://user-images.githubusercontent.com/67586/49587479-d00f6000-f918-11e8-9cf9-a3d477e9bc04.png)

Reply:
Thinking on this some more, I suggest I just land.  Until we all get unified behind a shared setting this churn will continue.  Once we unify, we could consider a sweeping PR to canonicalize...

Reply:
https://github.com/flutter/flutter-intellij/issues/2922 to follow-up.